client/src/components/PortfolioTable.tsx

```diff
@@ -598,7 +617,10 @@ export const PortfolioTable = React.memo((props: PortfolioTableProps) => {
                     {activeFilters.ownernamesActive && OwnernamesResultAlert}
                   </td>
                 </tr>
-              )}
+              )
+            ) : (
+              <></>
+            )}
```

Comment:
if you decide to go with the `!!` above

```suggestion
            )}
```